functions/extract-summary/main.py

```diff
@@ -27,8 +28,9 @@ def extract_data(url, bert_summary):
     # Not always article extracts correctly text from the HTML. In case text has not been extracted using alternative
     # way.
     if not article.text or len(article.text) < 100:
-        print("looks like article text is not extracted")
-        article.text = extractor.get_content(article.html)
+        print("looks like article text is not extracted, most likely download has failed")
```

Comment:
qq, have you tested this? I was under the impression that the library throws an error in case of any download problems. This is was here for cases when download is ok, however library failed to extract text from HTML.

Reply:
looks like you're partially right. I was under impression this if is for that fix but its not download but parse which fails. 

```
>>> url="https://www.forbes.com/sites/jackbrewster/2020/07/02/gop-suddenly-expresses-urgency-on-stimulus-congress-needs-to-act-in-july/#5867723172b5"
>>> article = Article(url)
>>> article.download()
>>> article.parse()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/newspaper/article.py", line 191, in parse
    self.throw_if_not_downloaded_verbose()
  File "/usr/local/lib/python3.7/site-packages/newspaper/article.py", line 532, in throw_if_not_downloaded_verbose
    (self.download_exception_msg, self.url))
newspaper.article.ArticleException: Article `download()` failed with 403 Client Error: Forbidden for url: https://www.forbes.com/sites/jackbrewster/2020/07/02/gop-suddenly-expresses-urgency-on-stimulus-congress-needs-to-act-in-july/#5867723172b5 on URL https://www.forbes.com/sites/jackbrewster/2020/07/02/gop-suddenly-expresses-urgency-on-stimulus-congress-needs-to-act-in-july/#5867723172b5
```

Reply:
I pushed tests, that covering exactly this problem. CI is not yet configured so you will have to manually run them:D